tests/auth.setup.ts

```diff
@@ -0,0 +1,38 @@
+import { expect, test as setup } from '@playwright/test'
+
+const BASE_URL = 'http://localhost:5173'
+const authFile1 = 'playwright/.auth/user1.json'
+// const authFile2 = 'playwright/.auth/user2.json'
+
+setup('authenticate as test1', async ({ page }) => {
+  await page.goto(`${BASE_URL}/login`)
+
+  // Fill in the username and password fields
+  await page.fill('input[name="email"]', 'test@capgo.app')
+  await page.fill('input[name="password"]', 'testtest')
+
+  // Click the submit button
+  await page.getByRole('button', { name: 'Log in' }).click()
+
+  // Expect the URL to change to the logged in dashboard
+  await expect(page).toHaveURL(`${BASE_URL}/app/home`, { timeout: 60_000 })
+
+  await page.context().storageState({ path: authFile1 })
+})
+
+// This will be usefull some day
+// setup('authenticate as test2', async ({ page }) => {
```

Comment:
Since setup is actually `test` from the playwright, you can uncomment this code and use `test.skip`

Reply:
I could also remove it. This is here only because I originally  started to work on this file for the [organization pr](https://github.com/Cap-go/capgo/pull/335/) and there having 2 users was required.

This is here so that if anyone wants to add a second user in the tests one day he or she can just uncoment this